changelog.d/20240118_114340_xavier.blanchot_add_flags_on_fix_available.md

```diff
@@ -0,0 +1,7 @@
+### Added
+
+- Adds two new flags for `ggshield sca scan` commands, `--ignore-fixable` and `--ignore-not-fixable` so that the user can filters the returned incidents depending on if incidents can be fixed or not. Both flags cannot be used simultaneously.
```

Comment:
```suggestion
- Adds two new flags for `ggshield sca scan` commands, `--ignore-fixable` and `--ignore-not-fixable` so that the user can filter the returned incidents depending on whether incidents can be fixed or not. Both flags cannot be used simultaneously.
```